pytest_dockerctl/__init__.py

```diff
@@ -62,7 +84,8 @@ def run(self, image, command=None, num=1, **kwargs):
             log.info("{}:{} Waiting on networking and health check...".format(
                 image, container.short_id))
             waitfor(container, ('NetworkSettings', 'IPAddress'))
-            waitfor(container, ('State', 'Health', 'Status'), expect='healthy')
+            if has_attr(container, ('State', 'Health', 'Status')):
```

Comment:
nice so this should avoid all the timeout issues for dockerfiles that don't have a check yah?

Reply:
Exactly, it actually throws `KeyError` as `waitfor` assumes `State/Health/Status` is always present in the container attributes.